adv_name.py

```diff
@@ -122,12 +123,12 @@ def get_captcha(driver):
 
 def main():
     options = Options()
-    DRIVER_PATH = '/usr/local/bin/chromedriver'
+    DRIVER_PATH = '/Users/sarvani/Downloads/chromedriver'
```

Comment:
Put your chrome driver also in that path.
In code it should not your home path only.
It cannot be used by others.